external/header.php

```diff
@@ -7,6 +7,19 @@
 
 include(dirname(__FILE__) . '/../xhprof_lib/config.php');
 
+function checkExtension()
```

Comment:
Please rename into `getExtensionName`, because it gets extension name, that is available.